kubernetes/spack/slack-spack-io/web/deployments.yaml

```diff
@@ -0,0 +1,54 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: slack-spack-io
+  namespace: spack
+  labels:
+    app: slack-spack-io
+    svc: web
+spec:
+  selector:
+    matchLabels:
+      app: slack-spack-io
+      svc: web
+  replicas: 2
+  template:
+    metadata:
+      labels:
+        app: slack-spack-io
+        svc: web
+    spec:
+      containers:
+      - name: web
+        image: "spack/slackin-extended:latest"
+        imagePullPolicy: Always
+        ports:
+        - name: http
+          containerPort: 80
+        env:
+        - name: SLACK_SUBDOMAIN
+          value: spackpm
+        - name: SLACKIN_ANALYTICS
+          value: "UA-101208306-3"
```

Comment:
Is this sensitive?  If so, you should move it to your secret.

Reply:
It’s not. Even if fetched from a secret it would be sent to the client to post to google.